src/db/tables/featureFlags.ts

```diff
@@ -38,24 +41,40 @@ export type FeatureFlagName =
 
 export async function getEnabledFeatureFlags(
   options:
-    | { ignoreAllowlist?: false; email: string }
-    | { ignoreAllowlist: true },
+    | { ignoreExperiments?: false; user: Session["user"] }
+    | { ignoreExperiments: true },
 ): Promise<FeatureFlagName[]> {
-  let query = knex("feature_flags")
+  const result = await knex("feature_flags")
     .select("name")
     .where("deleted_at", null)
     .and.where("expired_at", null)
     .and.where("is_enabled", true);
```

Comment:
nit: I guess this is technically "prior art", but looks like the only place where we're using the `.and.where(…)` syntax vs `.andWhere(…)`:

```sh
git grep -Pio "\.(and\.where|andWhere)\(" | sort | uniq -c

   1 scripts/collect-unresolved-breaches-addresses.js:.andWhere(
   1 src/db/tables/attributions.ts:.andWhere(
   2 src/db/tables/emailAddresses.js:.andWhere(
   5 src/db/tables/email_notifications.js:.andWhere(
   2 src/db/tables/featureFlags.ts:.and.where(
   2 src/db/tables/onerep_scans.ts:.andWhere(
   2 src/db/tables/subscribers.js:.andWhere(
```

Interesting, it looks like `knex.where()` also supports an [object syntax](https://knexjs.org/guide/query-builder.html#where):
```js
knex('users').where({
  first_name: 'Test',
  last_name:  'User'
}).select('id')
```
&hellip; although possibly only "interesting" to me since it looks like we use it about 10&times; throughout the codebase already, per <kbd>git grep -B1 -A2 -n "where({"</kbd>.